services/core-web/src/components/noticeOfWork/applications/review/ReviewNOWApplication.js

```diff
@@ -88,320 +98,362 @@ export const ReviewNOWApplication = (props) => {
+      </Row>
+      <Row gutter={16}>
+        <Col md={12} sm={24}>
+          {props.typeOfApplication !== "New Permit" && (
+            <>
+              <div className="field-title">
+                Annual Summary submitted for this site?
+                {props.isPreLaunch && <NOWFieldOriginTooltip />}
+                <NOWOriginalValueTooltip
+                  originalValue={props.renderOriginalValues("annual_summary_submitted").value}
+                  isVisible={props.renderOriginalValues("annual_summary_submitted").edited}
+                />
+              </div>
+              <Field
+                id="annual_summary_submitted"
+                name="annual_summary_submitted"
+                component={RenderRadioButtons}
+                disabled={props.isViewMode}
               />
-            </div>
-            <Field
-              id="description_of_land"
-              name="description_of_land"
-              component={RenderAutoSizeField}
-              disabled={props.isViewMode}
-              validate={[maxLength(4000)]}
-            /> */}
-            <ReviewApplicationFeeContent
-              initialValues={props.noticeOfWork}
-              isViewMode={props.isViewMode}
-              isAdmin={isAdmin}
-              proposedTonnage={props.proposedTonnage}
-              adjustedTonnage={props.adjustedTonnage}
-              proposedStartDate={props.proposedStartDate}
-              proposedAuthorizationEndDate={props.proposedAuthorizationEndDate}
-              isPreLaunch={props.isPreLaunch}
+            </>
+          )}
+        </Col>
+      </Row>
+      <Row gutter={16}>
+        <Col md={12} sm={24}>
+          <div className="field-title">
+            Permit Type
+            <NOWOriginalValueTooltip
+              originalValue={renderCodeValues(
+                props.permitTypeHash,
+                props.renderOriginalValues("application_permit_type_code").value
+              )}
+              isVisible={props.renderOriginalValues("application_permit_type_code").edited}
             />
-          </Col>
-          <Col md={12} sm={24}>
-            <div className="field-title">
-              Tracking Number
-              <NOWOriginalValueTooltip
-                originalValue={props.renderOriginalValues("now_tracking_number").value}
-                isVisible={props.renderOriginalValues("now_tracking_number").edited}
+          </div>
+          <Field
+            id="application_permit_type_code"
+            name="application_permit_type_code"
+            component={RenderSelect}
+            data={props.permitTypeOptions}
+            disabled={props.isViewMode}
+            validate={[validateSelectOptions(props.permitTypeOptions)]}
+          />
+        </Col>
+        <Col md={12} sm={24}>
+          {props.applicationPermitType === "MY-ABP" && (
+            <>
+              <div className="field-title">
+                Is this the first year of multi-year, area based application?
```

Comment:
"Is this the first year of multi-year, area based application?" --> "Is this the first year of a multi-year, area-based application?"

Reply:
fixed